tests/tx/test_verification.py

```diff
@@ -365,6 +440,51 @@ def test_transaction_verify(self) -> None:
         verify_sum_wrapped.assert_called_once()
         verify_reward_locked_wrapped.assert_called_once()
 
+    def test_transaction_validate_full(self) -> None:
```

Comment:
It's missing `verify_weight()` and `verify_parents_basic()`. They are both called from `verify_basic()`.

Reply:
Done in https://github.com/HathorNetwork/hathor-core/pull/824/commits/408b1d147c6e83216782bd3311bfa599604c6508

Reply:
Thanks!